lib/erasure/erasure.go

```diff
@@ -1,70 +1,102 @@
-// Copyright 2021 ChainSafe Systems (ON)
+// Copyright 2023 ChainSafe Systems (ON)
 // SPDX-License-Identifier: LGPL-3.0-only
 
 package erasure
 
+// #cgo CFLAGS: -I.
+// #cgo LDFLAGS: -L. -lerasure
+// #include "./rustlib.h"
+import (
+	"C"
+)
 import (
-	"bytes"
 	"errors"
-	"fmt"
-
-	"github.com/klauspost/reedsolomon"
+	"unsafe"
 )
 
-// ErrNotEnoughValidators cannot encode something for zero or one validator
-var ErrNotEnoughValidators = errors.New("expected at least 2 validators")
+var (
+	ZeroSizedData   = errors.New("Data can't be zero sized")
+	ZeroSizedChunks = errors.New("Chunks can't be zero sized")
```

Comment:
start with `Err` prefix also the error message should be lower case
```suggestion
	ErrZeroSizedData   = errors.New("data can't be zero sized")
	ErrZeroSizedChunks = errors.New("chunks can't be zero sized")
```